File: src/main/java/io/github/trainb0y/fabrihud/FabriHUD.java
package io.github.trainb0y.fabrihud;

import io.github.trainb0y.fabrihud.config.Config;
import net.fabricmc.api.ClientModInitializer;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public class FabriHUD implements ClientModInitializer {
    public static final Logger logger = LoggerFactory.getLogger("fabrihud");

    public void onInitializeClient() {
        Config.loadConfig();
    }
}

File: src/main/kotlin/io/github/trainb0y/fabrihud/FabriHUD.kt
package io.github.trainb0y.fabrihud

import io.github.trainb0y.fabrihud.config.Config.loadConfig
import net.fabricmc.api.ClientModInitializer
import org.slf4j.Logger
import org.slf4j.LoggerFactory

class FabriHUD : ClientModInitializer {
	override fun onInitializeClient() {
		loadConfig()
	}

	companion object {
		val logger: Logger = LoggerFactory.getLogger("io/github/trainb0y/fabrihud")
	}
}